lib/controllers/frontend/spree/apple_pay_domain_verification_controller.rb

```diff
@@ -0,0 +1,12 @@
+module Spree
+  class ApplePayDomainVerificationController < StoreController
+    skip_before_action :set_current_order, only: :show
+    include Spree::Core::ControllerHelpers
```

Comment:
Why do we need to include `ControllerHelpers` here?

Reply:
@damianlegawiec What's the reason of including `ControllerHelpers` here?

Reply:
@kushniryb we need for stuff like getting current Store etc - this will come in handy in the future when payment methods will be associated to Stores